docs/sources/reference/run.md

```diff
@@ -906,6 +906,25 @@ reference documentation.
 
 The following logging options are supported for this logging driver: [none]
 
+#### Logging driver: gelf
+
+Gelf logging driver for Docker. Writes log messages to gelf endpoint like Graylog or Logstash.
+`docker logs` command is not available for this logging driver
+
+The following logging options are supported for this logging driver:
+
+    --log-opt gelf-address=udp://host:port
+    --log-opt gelf-tag="database"
+
+`gelf-address` specifies the remote gelf server address where the driver connects to.
+Currently only `udp` is supported as transport and `port` must be specified.
```

Comment:
Currently only `udp` is supported as the transport and the `port` option must be specified.